### ISSUE TITLE
Python 3.11 support

### ISSUE THREAD

**Issue by kenodegard** [conda/conda#11170]
### Description

We need to investigate the changes necessary to support Python 3.11. Because this release cycle happens every year. It would also make sense for us to have a policy in place guiding the work that we need to regularly complete for this.

### Good to know

There is currently a rough draft of this document that could be used as a starting point for the work: 
https://github.com/conda-incubator/ceps/pull/24

```[tasklist]
- [ ] https://github.com/conda/conda/pull/12256
- [x] https://github.com/conda/conda/pull/12580
```

**Comment by EwoutH** [comment]
Python 3.11 is now in beta, which means no new features will be added.

- [Python 3.11.0b1 is now available](https://blog.python.org/2022/05/python-3110b1-is-now-available.html)
- [What’s New In Python 3.11](https://docs.python.org/3.11/whatsnew/3.11.html)

**Comment by EwoutH** [comment]
Any update on this effort? Python 3.11 is now in the release candidate phase and will be released in two weeks. Many projects already support Python 3.11 and have wheels available, including NumPy, Pandas, SciPy, Matplotlib and Seaborn.

**Comment by travishathaway** [comment]
@EwoutH, thanks for checking in. This issue is high on our priority list, and we hope to begin working on it in an upcoming sprint. I can not say for certain when this will be completed though.

**Comment by EwoutH** [comment]
Thanks for getting back! Great to hear the team is working on this, keep us in the loop :)

**Comment by johnnynunez** [comment]
Yeah! Python 3.11 support please. It should be the default.
Python 3.11 is between 10-60% faster than Python 3.10. On average, we measured a 1.25x speedup on the standard benchmark suite. See Faster CPython for details.
https://docs.python.org/3.11/whatsnew/3.11.html
https://www.phoronix.com/news/Python-3.11-Released

**Comment by johnnynunez** [comment]
Benchmark https://www.phoronix.com/review/python-311-performance

**Comment by EwoutH** [comment]
Hi @travishathaway! Do you have anything to share as of this moment?

**Comment by EwoutH** [comment]
We're now over 4 months over rc1 and almost two months after stable release. Could we get an update on 3.11 support?

**Comment by johnnynunez** [comment]
could you build all packages for python 3.11 in channel python 3.11? https://github.com/pytorch/vision/pull/7108
![image](https://user-images.githubusercontent.com/22727137/213490577-f7b56fb3-c9fd-4372-95d8-3bf49aaab892.png)

**Comment by travishathaway** [comment]
@johnnynunez, unfortunately this is not the appropriate place for that conversation. These are decisions that the channel owners themselves need to make. For suggestions on ways to improve packages for the `defaults` channel, please file an issue here: https://github.com/ContinuumIO/anaconda-issues/issues/new/choose .

@EwoutH, just to give you a brief update, the status of this issue is largely dependent on those who maintain the channels where conda is available from. Right now these channels include `defaults` which is maintained by Anaconda and `conda-forge` which is maintained by conda-forge.  The core development team is going to try its best to have Python 3.11 support ready by the March release, but we cannot make any promises.

I really appreciate everyone's patience on this! I will post here again once I have more updates.

**Comment by RNarayan73** [comment]
Due to be released on March 14, fingers crossed!
[https://www.anaconda.com/blog/upcoming-releases-anaconda-distribution-2023-03-and-beyond#:~:text=Python%203.11%20support%20will%20be,of%20Anaconda's%20biggest%20releases%20yet!](url)

**Comment by rickyyx** [comment]
Didn't see relevant info of 3.11 in the release note: https://github.com/conda/conda/releases/tag/23.3.0

But are we having 3.11 support officially?

**Comment by johnnynunez** [comment]
When will the python3.11 image be available for testing?

**Comment by kenodegard** [comment]
Python 3.11 support added via #12256, this will be included in the May release of `conda 23.5.0`

**Comment by EwoutH** [comment]
Awesome, great to hear!

Maybe this process can now be documented, while it's still fresh in everyones memory, so that it doesn't take so long for future Python releases.
